### ISSUE TITLE
PDFs Lack /Title

### ISSUE THREAD

**Issue by chadhogg** [latex3/tagging-project#1233]
I am not certain that what I am reporting is *exactly* related to tagging, but it is related to creating PDF files that meet WCAG 2.1.

My institution uses a tool known as YuJa Panorama to examine documents uploaded to our Learning Management System and assign them an accessibility score.  Every PDF file that I upload receives a complaint that it labels as "The title is missing from the document's properties."  These documents do, in fact, have titles.  Said titles are visible if you open the document in Adobe Acrobat and view its document properties.  And Adobe's built-in accessibility checker says that the documents do have an appropriate title.

It seems quite likely that this is a false positive from YuJa Panorama, but I would like to gather some more information before definitively concluding this.  Yuja Panorama offers users the ability remediate the problem directly in the LMS.  Doing so modifies the document in many ways, but in particular it adds a section like this:

```
11 0 obj
/Title (The Fake Title Added By YuJa)
endobj
```

And it seems noteworthy that this is precisely what is suggested in the WCAG documentation: https://www.w3.org/WAI/WCAG22/Techniques/pdf/PDF18

This leaves me with two plausible explanations:

1: My initial documents really lack this feature, and hopefully this can be fixed in a future update.
2: My initial documents have this feature encoded in a different way that my accessibility checker (and I, manually) is not able to find.  The accessibility checker needs to be updated.

I am hoping that it is easy for someone more familiar with the PDF format than I am to distinguish between these.

Here's my source file"

```
\DocumentMetadata{
  lang = en,
  pdfstandard = ua-2,
  pdfstandard = a-4f,
  tagging = on,
  tagging-setup = {math/setup=mathml-SE} 
}
\documentclass{article}
\title{The Actual Title In The Document}
\begin{document}
  \maketitle
  Lorem ipsum.
\end{document}

```

Attached files:
- Minimum-Borken-Example.pdf is the result of processing that source through lualatex 1.22.0 development ID 7673 with all packages updated through tlmgr on 2026-02-15.
- fixed.pdf is the result of applying YuJa Panorama's fix to Minimum-Broken-Example.pdf

[Minimum-Broken-Example.pdf](https://github.com/user-attachments/files/25328442/Minimum-Broken-Example.pdf)

[fixed.pdf](https://github.com/user-attachments/files/25328443/fixed.pdf)
